recipes/openjpeg/all/conanfile.py

```diff
@@ -142,7 +143,7 @@ def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "OpenJPEG")
         self.cpp_info.set_property("cmake_target_name", "openjp2")
         self.cpp_info.set_property("cmake_build_modules", [self._module_vars_rel_path])
-        self.cpp_info.set_property("pkg_config_name", "libopenjp2")
+        self.cpp_info.set_property("pkg_config_name", "openjp2")
```

Comment:
It actually is `libopenjp2.pc`: 

https://c3i.jfrog.io/c3i/misc-v2/logs/pr/22908/2-linux-gcc/openjpeg/2.5.1//b647c43bfefae3f830561ca202b6cfd935b56205-build.txt

```
openjpeg/2.5.1: RUN: cmake --install "/home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/b/build/Release" --prefix "/home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/p"
-- Install configuration: "Release"
-- Installing: /home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/p/lib/cmake/openjpeg-2.5/OpenJPEGTargets.cmake
-- Installing: /home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/p/lib/cmake/openjpeg-2.5/OpenJPEGTargets-release.cmake
-- Installing: /home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/p/lib/cmake/openjpeg-2.5/OpenJPEGConfig.cmake
-- Installing: /home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/p/lib/cmake/openjpeg-2.5/OpenJPEGConfigVersion.cmake
-- Installing: /home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/p/lib/pkgconfig/libopenjp2.pc
-- Installing: /home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/p/include/openjpeg-2.5/opj_config.h
-- Installing: /home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/p/lib/libopenjp2.a
-- Installing: /home/conan/w/prod-v2/bsr/98187/addad/p/b/openj494439aedf2e3/p/include/openjpeg-2.5/openjpeg.h
```

Same for AUR (Arch Linux): https://archlinux.org/packages/extra/x86_64/openjpeg2/files/


Also, could you please remove the line 162:

`self.cpp_info.names["pkg_config"] = "libopenjp2"`

Reply:
Ah, do mean the attribute `name` in the PC file? I guess it does not matter, because we will use something like `pkg-config --cflags --libs libopenjp2`

Reply:
@uilianries 
I misunderstood.
Thanks! Let me revert it.